docs/release-notes/2.25.0.md

```diff
@@ -0,0 +1,88 @@
+# Percona Monitoring and Management 2.25.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">December 7, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+Highlights. 
+** (Technology Preview ) Ability to connect your Percona Monitoring and Management instances to your Percona Portal to get the latest intelligence from Percona and have all your accounts and relations with Percona centralized. READ_MORE

```

Comment:
Better to use Percona Platform, or Percona Platform Portal instead of Percona Portal